tools.go

```diff
@@ -0,0 +1,7 @@
+// +build tools
```

Comment:
In this repo, we follow https://github.com/golang-standards/project-layout.

Additional tools should be in `tools` folder instead of `tools.go`.